website/test/e2e/api_spec.js

```diff
@@ -133,4 +133,11 @@ describe('Api', function() {
     expect(browser.getCurrentUrl()).
         toMatch(/api\?view=webdriver.WebDriver.prototype.executeAsyncScript/);
   });
+
+  it('should sort the menu to put webdriver docs next to the relavant ' +
```

Comment:
nit: relevant.

Why don't we just assert expect(apiPage.getAdultNmaes()).toBe(['a', 'b', ..., 'browser', 'webdriver.WebDriver', ..., 'y', 'z'])
So we can assert all the orders?